layouts/_default/single.html

```diff
@@ -178,54 +178,116 @@ <h2 class="">Your ballot:</h2>
               <div class="mx-auto mt-2 grid gap-4 sm:grid-cols-2">
                 {{ $cardClass := `rounded border border-g-5 p-3` }}
                 {{ $hedClass := `mt-2 uppercase font-semibold text-g-7` }}
+
+
                 <div class="{{ $cardClass }}">
-                  <h3 class="{{ $hedClass }}">Governor</h3>
+                  <h3 class="{{ $hedClass }}">Justice of the Supreme Court</h3>
                   <ul>
-                    <template
-                      x-for="candidate of shuffled('governor')"
-                      key="name"
-                    >
-                      {{ block "candidate" . }}
-                        <li class="">
-                          <span
-                            :style="{ backgroundColor: $partyColor(candidate.party)}"
-                            class="mb-px inline-block h-3 w-3 rounded-full"
-                          ></span>
-                          <span
-                            class="font-bold"
-                            x-text="candidate.name"
-                          ></span>
-                          (<span x-text="$party(candidate.party)"></span>)
-                        </li>
-                      {{ end }}
-                    </template>
+                    <li>
+                      <span
+                        style=" background: rgb(0, 0, 255)"
```

Comment:
Why is this an inline style instead of just bg-white?

Reply:
I thought it best to keep consistent with the dynamically generated ones that pull color from political party

<img width="46" alt="Screenshot 2023-04-19 at 10 57 44 AM" src="https://user-images.githubusercontent.com/818602/233116256-b8b263b2-cb6f-4ef2-ba10-2bcf451203b0.png">